### PR TITLE
WRN-3840: Migrate MoonstoneDecorator and VirtualList unit tests to Testing Library. 

### PR DESCRIPTION
**Pull request by alexandrumorariu**
### Checklist
* [X] I have read and understand the contribution guide
* [ ] A CHANGELOG entry is included
* [ ] At least one test case is included for this feature or bug fix
* [X] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
Migrated tests from Enzyme to Testing Library.

### Resolution
Changed test structure to work with Testing Library.

### Links
WRN-3840

### Comments

### CODE REVIEW

File: MoonstoneDecorator/tests/MoonstoneDecorator-specs.js [hunk 1]
```diff
@@ -1,85 +1,69 @@
...
 		const App = MoonstoneDecorator(config, AppRoot);
-		const subject = mount(
-			<App className="author-class" />
-		);
+		render(<App className="author-class" data-testid="app" />);
 
 		Spotlight.terminate();
 
-		const appRoot = subject.find('[data-app]');
+		const appRoot = screen.getByTestId('app');
```
Comment (adrian-cocoara-lgp) [inline MoonstoneDecorator/tests/MoonstoneDecorator-specs.js]: I think you can remove this empty space

File: MoonstoneDecorator/tests/MoonstoneDecorator-specs.js [hunk 2]
```diff
@@ -1,85 +1,69 @@
...
+		const config = {ri: false, i18n: false, spotlight: false, float: true, overlay: false};
+		const App = MoonstoneDecorator(config, AppRoot);
+		render(<App data-testid="app" />);
 
-			const appRoot = subject.find('[data-app]');
+		Spotlight.terminate();
 
-			const expected = false;
-			const actual = appRoot.hasClass('moonstone');
+		const appRoot = screen.getByTestId('app');
```
Comment (adrian-cocoara-lgp) [inline MoonstoneDecorator/tests/MoonstoneDecorator-specs.js]: remove empty space

File: MoonstoneDecorator/tests/MoonstoneDecorator-specs.js [hunk 3]
```diff
@@ -1,85 +1,69 @@
...
+		const config = {ri: false, i18n: false, spotlight: false, float: false, overlay: true};
+		const App = MoonstoneDecorator(config, AppRoot);
+		render(<App data-testid="app" />);
 
-			const appRoot = subject.find('[data-app]');
+		Spotlight.terminate();
 
-			const expected = false;
-			const actual = appRoot.hasClass(css.bg);
+		const appRoot = screen.getByTestId('app');
```
Comment (adrian-cocoara-lgp) [inline MoonstoneDecorator/tests/MoonstoneDecorator-specs.js]: remove empty space

File: MoonstoneDecorator/tests/MoonstoneDecorator-specs.js (hunk 1 above)
Comment (alexandrumorariu) [reply MoonstoneDecorator/tests/MoonstoneDecorator-specs.js]: Done

**Review by adrian-cocoara-lgp — APPROVED** [review]
LGTM

File: VirtualList/tests/VirtualList-specs.js [hunk 4]
```diff
@@ -263,128 +242,129 @@ describe('VirtualList', () => {
...
-					/>
-				);
+		test('should set `aria-label` to previous scroll button in the horizontal scrollbar', () => {
+			const label = 'custom button aria label';
+			render(
+				<VirtualList
+					clientSize={clientSize}
+					dataSize={dataSize}
+					direction="horizontal"
+					scrollLeftAriaLabel={label}
```
Comment (stanca-pop-lgp) [inline VirtualList/tests/VirtualList-specs.js]: Please sort props.

**Review by stanca-pop-lgp — APPROVED** [review]
LGTM

**Pull request merged**